multiapps-common/src/main/java/org/cloudfoundry/multiapps/common/util/MapUtil.java

```diff
@@ -41,33 +40,18 @@ private MapUtil() {
         return result;
     }
 
-    public static <K, V> Map<K, V> asMap(K key, V value) {
-        Map<K, V> result = new TreeMap<>();
-        result.put(key, value);
-        return result;
-    }
```

Comment:
Why this method is gone?
```
[ERROR] /home/travis/build/cloudfoundry-incubator/multiapps/multiapps-common/src/test/java/org/cloudfoundry/multiapps/common/util/JsonUtilTest.java:[51,34] cannot find symbol
  symbol:   method asMap(java.lang.String,java.util.Map<java.lang.String,java.lang.Object>)
  location: class org.cloudfoundry.multiapps.common.util.MapUtil
```

Reply:
Because there are a lot of factory methods that replace it: `Map.of`